jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/CheckboxTests.java

```diff
@@ -0,0 +1,84 @@
+package io.github.epam.material.tests.inputs;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.checkboxPage;
+import static io.github.com.MaterialNavigator.openSection;
+
+public class CheckboxTests extends TestsInit {
+
+    private void checkboxTestLogic(String section, int index) {
+        if (checkboxPage.checkbox.isEnabled(section, index)) {
+            checkboxPage.checkbox.check(section, index);
+            checkboxPage.checkbox.is().checked(section, index);
+            checkboxPage.checkbox.uncheck(section, index);
+            checkboxPage.checkbox.is().unChecked(section, index);
+        } else
+            checkboxPage.checkbox.is().disabled(section, index);
+    }
+
+    private void textTestLogic(String section, int index) {
+        checkboxPage.checkbox.is().checkText(section, index);
+    }
+
+    private void groupTestLogic(String section, int index) {
+        int secondIndex = (index + 3) / 7 > 0 ? (index + 3) % 7 + 1 : (index + 3) % 7;
+        checkboxPage.checkbox.check(section, index);
+        checkboxPage.checkbox.is().checked(section, index);
+        checkboxPage.checkbox.is().checked(section, secondIndex);
+        checkboxPage.checkbox.uncheck(section, index);
+        checkboxPage.checkbox.is().unChecked(section, index);
+        checkboxPage.checkbox.is().unChecked(section, secondIndex);
+    }
+
+    private void groupTestErrorLogic(String section) {
+        checkboxPage.checkbox.check(section, 1);
+        checkboxPage.checkbox.is().hasError();
+        checkboxPage.checkbox.check(section, 2);
+        checkboxPage.checkbox.is().hasNoError();
+    }
+
+    @Test
+    public void BasicCheckboxTest() {
+        openSection("Checkboxes");
+        checkboxPage.shouldBeOpened();
+        String section = "Basic";
+        for (int i = 1; i < 9; i++)
+            checkboxTestLogic(section, i);
```

Comment:
Please check details of the checkboxes: e.g. that it's have class "MuiCheckbox-colorSecondary" or "Mui-checked". The same for other tests

Reply:
Done